torch/csrc/autograd/functions/pybind.h

```diff
@@ -11,21 +11,4 @@ namespace py = pybind11;
 
 namespace pybind11 { namespace detail {
 
-// handle Python <-> torch::autograd::Function conversions
```

Comment:
I'm not actually sure if this is used anywhere, but it doesn't seem to cause anything to fail when I delete it.